lang/java/avro/src/test/java/org/apache/avro/specific/TestRecordWithLogicalTypes.java

```diff
@@ -0,0 +1,697 @@
+/**
+ * Autogenerated by Avro
+ * 
+ * DO NOT EDIT DIRECTLY
+ */
+package org.apache.avro.specific;
+@SuppressWarnings("all")
+@org.apache.avro.specific.AvroGenerated
+public class TestRecordWithLogicalTypes extends org.apache.avro.specific.SpecificRecordBase implements org.apache.avro.specific.SpecificRecord {
+  private static final long serialVersionUID = -4211233492739285532L;
+  public static final org.apache.avro.Schema SCHEMA$ = new org.apache.avro.Schema.Parser().parse("{\"type\":\"record\",\"name\":\"TestRecordWithLogicalTypes\",\"namespace\":\"org.apache.avro.specific\",\"fields\":[{\"name\":\"b\",\"type\":\"boolean\"},{\"name\":\"i32\",\"type\":\"int\"},{\"name\":\"i64\",\"type\":\"long\"},{\"name\":\"f32\",\"type\":\"float\"},{\"name\":\"f64\",\"type\":\"double\"},{\"name\":\"s\",\"type\":[\"null\",\"string\"],\"default\":null},{\"name\":\"d\",\"type\":{\"type\":\"int\",\"logicalType\":\"date\"}},{\"name\":\"t\",\"type\":{\"type\":\"int\",\"logicalType\":\"time-millis\"}},{\"name\":\"ts\",\"type\":{\"type\":\"long\",\"logicalType\":\"timestamp-millis\"}}]}");
+  public static org.apache.avro.Schema getClassSchema() { return SCHEMA$; }
+  @Deprecated public boolean b;
+  @Deprecated public int i32;
+  @Deprecated public long i64;
+  @Deprecated public float f32;
+  @Deprecated public double f64;
+  @Deprecated public java.lang.CharSequence s;
+  @Deprecated public org.joda.time.LocalDate d;
```

Comment:
@rdblue: Sorry for belated response. Ok, I'm glad that you see a way for configuring between joda and java. Not clear to me yet how, but I will take a look at the -string option as an example. Absolutelly, go ahead and submit. WRT contributions, I quite busy with what i do, but i also understand there is no much rush, so perhaps I will just start a longer-living fork and experiment for half an hour every other day.